i18n/locales/en/common.json

```diff
@@ -311,6 +313,7 @@
   "Password": "Password",
   "Paste": "Paste",
   "Pending...": "Pending...",
+  "Percentage of successfully forged blocks of when the delegate should have forged a block of transactions.": "Percentage of successfully forged blocks of when the delegate should have forged a block of transactions.",
```

Comment:
If I am not wrong this is not properly write, should be without the **of** between blocks and when, like this.

Percentage of successfully forged blocks **of** when the delegate should have forged a block of transactions
Percentage of successfully forged blocks when the delegate should have forged a block of transactions

Reply:
Checked all tooltips with marketing team